packages/itwinui-react/src/core/ExpandableBlock/ExpandableBlock.tsx

```diff
@@ -101,6 +110,10 @@ export const ExpandableBlock = React.forwardRef((props, ref) => {
       return;
     }
 
+    if (disabled) {
```

Comment:
This if statement can be combined with the if statement above it

Reply:
done